server/storage/types.go

```diff
@@ -21,6 +21,7 @@ type (
 const (
 	ClusterStateEmpty ClusterState = iota + 1
 	ClusterStateStable
+	ClusterStatePrepare
```

Comment:
Is this state not persisted?

Reply:
I used a state that pb had before, which corresponds to `ClusterView_PREPARE_REBALANCE`.